pkg/vulnsrc/python/python_test.go

```diff
@@ -0,0 +1,75 @@
+package python
+
+import (
+	"os"
+	"reflect"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy-db/pkg/db"
+	"github.com/aquasecurity/trivy-db/pkg/vulnsrc/vulnerability"
+)
+
+func TestVulnSrc_Update(t *testing.T) {
+	type args struct {
+		dir string
+	}
+	testCases := []struct {
+		name string
+		args struct {
+			dir string
+		}
+		expectedAdvisory []types.Advisory
+		expectErr bool
+	}{
+		{
+			name: "happy path",
+			args: args{
+				dir: "testdata/fixtures",
+			},
+			expectedAdvisory: []types.Advisory {
+				{
+					VulnerabilityID: "CVE-2018-9986",
+				},
```

Comment:
I may be missing something, but why doesn't it include `Spec`?

Reply:
@knqyf263 can you please help explain what is `Spec`?